### PR TITLE
Fix error when data is an empty object

### PR DESCRIPTION
**Pull request by Adonais0**
The `GradeDistribution` page and `AssignmentPlanning` page shows error when `gradeData` or `assignmentData` is an empty object.

### CODE REVIEW

File: assets/src/containers/AssignmentPlanning.js [hunk 1]
```diff
@@ -66,6 +66,15 @@ function AssignmentPlanning (props) {
     />
   )
 
+  const tableBuilder = (loaded, assignmentData, table) => {
```
Comment (justin0022) [inline assets/src/containers/AssignmentPlanning.js]: `table` param isn't used - could you please remove it?

File: assets/src/containers/AssignmentPlanning.js [hunk 2]
```diff
@@ -88,7 +97,7 @@ function AssignmentPlanning (props) {
                   <MenuItem value={75}>75%</MenuItem>
                 </Select>
               </FormControl>
-              {loaded ? AssignmentTable(assignmentData.plan) : <Spinner />}
+              { tableBuilder(loaded, assignmentData) }
```
Comment (justin0022) [inline assets/src/containers/AssignmentPlanning.js]: Just curious, why provided `tableBuilder` with the `loaded` variable? 

If we make it `{loaded ? tableBuilder(assignmentData.plan) : <Spinner />}`? That way, `tableBuilder` is only called when the data is loaded. As written, `tableBuilder` is called while data is loading. Not a huge deal though, just a thought!

Comment (Adonais0) [reply assets/src/containers/AssignmentPlanning.js]: Sure, done.

File: assets/src/containers/AssignmentPlanning.js (hunk 1 above)
Comment (Adonais0) [reply assets/src/containers/AssignmentPlanning.js]: Yep, removed

File: assets/src/containers/AssignmentPlanning.js (hunk 2 above)
Comment (justin0022) [reply assets/src/containers/AssignmentPlanning.js]: Perfect, looks good to me!

**Pull request merged**